### PR TITLE
Authorize

### PR DESCRIPTION
**Pull request by gregory-latinier**
Feature: Design and text UI for authorizing an app

### CODE REVIEW

**Comment by bonustrack** [comment]
There is 2 style of buttons:
image
image
There should be only the second one style.

**Comment by bonustrack** [comment]
@gregory-latinier Can you change "Offline" to "Offline Access" and also i think we can remove the sentence about the 7 days. As far as i can see, others app don't tell about when access_token expire. It's just implicit that it will expire if there is no "offline access" granted.

File: src/components/Auth/Authorize.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default class Authorize extends Component {
       if (scope) {
         scopes = scope
           .split(',')
-          .filter(o => config.authorized_operations.includes(o) || o === 'offline');
+          .filter(o => config.authorized_operations.includes(o) || o === 'offline_access');
```
Comment (bonustrack) [inline src/components/Auth/Authorize.js]: The label should be "Offline Access" but the scope should stay `offline`.

Comment (gregory-latinier) [reply src/components/Auth/Authorize.js]: Oh ok I miss understood that

**Pull request merged**